### DISCUSSION TITLE
Contour lines in default render themes

### DISCUSSION THREAD

**Discussion by Sublimis** [mapsforge/mapsforge#1519] [Q&A] [answered]
We noticed there are no contour lines displayed in MotoRider, Mapsforge, nor the Osmarender themes included with the library. Can we get this implemented?

**Comment by devemux86** [comment]
Mapsforge maps do not contain contour lines, so the themes do not have such rules.

We could probably include them to work with OpenAndroMaps or Freizeitkarte maps,
although these maps have their own themes.

**Motorider** theme is developed by @MotoUKRider and **Cruiser** community and I merge it in Mapsforge:
- https://github.com/devemux86/cruiser/discussions/612

Can you please write your suggestions there?

**Comment by devemux86** [comment, marked as answer]
Motorider theme now includes contour lines in the layers menu:

![cruiser-contour-lines-2](https://github.com/user-attachments/assets/3c15003e-3501-4803-a71d-451f19a1ef91)
